lux/core/series.py

```diff
@@ -211,7 +215,83 @@ def recommendation(self):
             self.name = " "
         ldf = LuxDataFrame(self)
 
-        if self._recommendation is not None and self._recommendation == {}:
-            ldf.maintain_metadata()
-            ldf.maintain_recs()
+        ldf.maintain_metadata()
+        ldf.maintain_recs()
         return ldf._recommendation
+
+    @property
+    def exported(self) -> Union[Dict[str, VisList], VisList]:
```

Comment:
Can we reuse and share this code with the `exported` property function in `frame.py`?

Reply:
got it!